### PR TITLE
feat(sign in with apple): Upgrade to kotlin 1.6.10

### PR DESCRIPTION
**Pull request by HenriBeck**

### CODE REVIEW

**Comment by deathcoder** [comment]
flutter is now a 2.10 can this be merged? or is something missing?

**Comment by nerder** [comment]
Something is missing here? Can this be merged?

File: packages/sign_in_with_apple/sign_in_with_apple/example/android/build.gradle [hunk 1]
```diff
@@ -21,6 +21,8 @@ allprojects {
 rootProject.buildDir = '../build'
 subprojects {
     project.buildDir = "${rootProject.buildDir}/${project.name}"
+}
```
Comment (tp) [inline packages/sign_in_with_apple/sign_in_with_apple/example/android/build.gradle:24]: Does this have any (desired) effect?

File: packages/sign_in_with_apple/sign_in_with_apple/android/build.gradle [hunk 2]
```diff
@@ -2,7 +2,7 @@ group 'com.aboutyou.dart_packages.sign_in_with_apple'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.5.31'
```
Comment (tp) [inline packages/sign_in_with_apple/sign_in_with_apple/android/build.gradle:5]: How come this now points to 1.5, when initially it was supposed to be 1.6?

Comment (HenriBeck) [reply packages/sign_in_with_apple/sign_in_with_apple/android/build.gradle:5]: Initially for 2.10 only `1.5.31` was the minimum required version as per https://docs.flutter.dev/release/breaking-changes/kotlin-version

File: .github/workflows/dart.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ jobs:
           java-version: "12.x"
       - uses: subosito/flutter-action@4389e6cbc6cb8a4b18c628ff96ff90be0e926aa8 # v1.5.3
         with:
-          flutter-version: "2.5.1"
+          flutter-version: "2.5.3"
```
Comment (HenriBeck) [inline .github/workflows/dart.yml:35]:
```suggestion
          flutter-version: "2.8.1"
```

**Review by tp — APPROVED** [review]

**Comment by tp** [comment]
@HenriBeck So should we release this on pub.dev, or do you have any further change in mind?

**Comment by nerder** [comment]
Great @HenriBeck

**Pull request merged**